assets/js/base/components/store-notices-container/store-notices-container.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Notice } from 'wordpress-components';
+import { useStoreNoticesContext } from '@woocommerce/base-context';
+import { filter } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const getWooClassName = ( { status = 'default' } ) => {
+	switch ( status ) {
+		case 'error':
+			return 'woocommerce-message woocommerce-error';
+		case 'success':
+			return 'woocommerce-message woocommerce-success';
+		case 'info':
+		case 'warning':
+			return 'woocommerce-message woocommerce-info';
+	}
+	return '';
+};
+
+const StoreNoticesContainer = ( { className, notices } ) => {
+	const { removeNotice } = useStoreNoticesContext();
+	const wrapperClass = classnames( className, 'wc-block-components-notices' );
+
+	if ( ! notices.length ) {
+		return null;
+	}
+
+	const regularNotices = filter(
+		notices,
+		( notice ) => notice.type !== 'snackbar'
+	);
+
```

Comment:
The issue with scroll jumping when a snackbar appeared or disappeared came from here: after the `filter` we need to return if `regularNotices.length === 0`. Otherwise, we are rendering an empty `.wc-block-components-notices`.

Reply:
Yup another good catch! Thanks :)